### DISCUSSION TITLE
Is there a way to remove transparency from lualine?

### DISCUSSION THREAD

**Discussion by uchoa** [nvim-lualine/lualine.nvim#1304] [Configuration]
I'm using catppuccin/nvim as a colorscheme with `transparent_background = true`.  Is there a way to have a non transparent lualine despite of this colorscheme configuration?

**Comment by iton0** [comment]
You might be able to do this by explicitly setting the theme of your lualine. However, I believe since your colorscheme changes the highlights for your lualine, you may have to do this manually in your colorscheme config.
